### DISCUSSION TITLE
Copy&paste modifiers from one behaviour to another

### DISCUSSION THREAD

**Discussion by elisaffueyo** [olivierfriard/BORIS#720] [Q&A]
Hi,

In my ethogram, I have behaviours that share the exact same sets of modifiers, is it possible to transfer modifiers from one behaviour to another?

Best,

Elisa

**Comment by olivierfriard** [comment]
Hi,
it is not possible to copy/paste the modifiers from a behavior to an another behavior (for now) but you can set a first behavior with modifiers and clone it (Behavior button > Clone behavior).
